data/icons/meson.build

```diff
@@ -0,0 +1,22 @@
+icon_dirs = [
+  '16x16',
+  '24x24',
+  '32x32',
+  '48x48',
+  '256x256',
+  '512x512',
+]
+
+foreach i:icon_dirs
+  install_data(
+    join_paths(i, 'gnome-pomodoro.png'),
+    install_dir: join_paths(get_option('prefix'), get_option('datadir'), 'icons', 'hicolor', i, 'apps'),
```

Comment:
I would at least add `hicolor_icondir = install_dir: join_paths(get_option('prefix'), get_option('datadir'), 'icons', 'hicolor'` if not using the datadir. This is really long and hard to read.